node_modules/gh-users/tests/util.js

```diff
@@ -297,6 +298,73 @@ var assertUpdateAdminStatusFails = module.exports.assertUpdateAdminStatusFails =
 };
 
 /**
+ * Assert that a user's password can be changed
+ *
+ * @param  {RestClient}         client                          The REST client to make the request with
+ * @param  {Number}             id                              The id of user for which to change the local password
+ * @param  {String}             newPassword                     The new password for the user
+ * @param  {String}             [oldPassword]                   The previous password for the user. This is only required when the current user is not an administrator
+ * @param  {Function}           callback                        Standard callback function
+ * @param  {RestClient}         callback.client                 A client that's authenticated with the new password
+ * @throws {AssertionError}                                     Error thrown when an assertion failed
+ */
+var assertChangePassword = module.exports.assertChangePassword = function(client, id, newPassword, oldPassword, callback) {
+    // Change the user's password
+    client.user.changePassword(id, newPassword, oldPassword, function(err) {
+        assert.ok(!err);
+
+        // Grab the user's profile information so we can attempt a local login
+        // and verify the password was changed successfully
+        assertGetUser(client, id, function(user) {
+
+            // Get the object of the application the user belongs to so we can create a REST client
+            TestsUtil.getGlobalAdminRestClient(function(globalAdminClient) {
+                AppTestsUtil.assertGetApp(globalAdminClient, user.AppId, null, function(app) {
```

Comment:
http://www.imdb.com/title/tt0120772/

I think we can just say `Get the application the user belongs to...`